docs/docs/quickstart.md

```diff
@@ -12,18 +12,22 @@ If you arrived here by chance and you're curious as to what Kurtosis _is_, [see
 
 If you're ready to get going, this guide will take ~15 minutes and will walk you through building a basic Kurtosis package. The package that you build will start a Postgres server, seed it with data, put an API in front of it, and automate loading data into it.
 
-You need to [have Kurtosis installed][installing-kurtosis-guide] (or [upgraded to latest][upgrading-kurtosis-guide] if you already have it), but you do not need any other knowledge.
+#### Setup
+Before you proceed, make sure you have [Kurtosis installed][installing-kurtosis-guide] (or [upgraded to latest][upgrading-kurtosis-guide] if you already have it) and Docker is started and a Docker daemon is running. Other than these two items, you do not need any other knowledge.
```

Comment:
```suggestion
Before you proceed, make sure you have [Kurtosis installed][installing-kurtosis-guide] (or [upgraded to latest][upgrading-kurtosis-guide] if you already have it), Docker is started, and the Docker engine is running.
```
Think this can be trimmed